_pkgdown.yml

```diff
@@ -1,3 +1,5 @@
+url: https://pharmaverse.github.io/admiral/index.html
```

Comment:
Could you remove `index.html`?

I have built the site and now the search result refers to https://pharmaverse.github.io/admiral/index.html/reference/derive_param_first_event.html?q=first_e#null

Reply:
`url: https://pharmaverse.github.io/admiral`  OR `url: https://pharmaverse.github.io/admiral/`

Reply:
I tried `url: https://pharmaverse.github.io/admiral/` and this seems to work.

Reply:
I also tried without slash. It works as well.